### ISSUE TITLE
instance_server_ca_cert sensitve output error

### ISSUE THREAD

**Issue by YpNo** [terraform-google-modules/terraform-google-sql-db#567]
### TL;DR

A plan/apply generates errors because of sensitive output about the instance_server_ca_cert.

### Expected behavior

Plan / Apply without error

### Observed behavior

 Error: Output refers to sensitive values
│ 
│   on outputs.tf line 53:
│   53: output "instance_server_ca_cert" {
│ 
│ To reduce the risk of accidentally exporting sensitive data that was
│ intended to be only internal, Terraform requires that any root module
│ output containing sensitive data be explicitly marked as sensitive, to
│ confirm your intent.
│ 
│ If you do intend to export this data, annotate the output value as
│ sensitive by adding the following argument:
│     sensitive = true
╵
ERRO[0012] terraform invocation failed in ./cloudsql/instances/.terragrunt-cache/vnEX4_xXUD6Ptk0lLrFRumQoK4s/hOxGJB_iOktveSaSjKRYVjS_vcY/modules/postgresql

### Terraform Configuration

```hcl
# Terragrunt usage
...
terraform {
  source = "tfr:///GoogleCloudPlatform/sql-db/google//modules/postgresql?version=18.2.0"
}

# Indicate the input values to use for the variables of the module.
inputs = {
  project_id = dependency.metadata.outputs.project_id

  # Instance
  name             = "instance-1"
  database_version = "POSTGRES_11"

  # Master Configuration
  tier                    = "db-f1-micro"
  zone                    = "europe-west9-b"
  availability_type       = "ZONAL"
  maintenance_window_day  = 2
  maintenance_window_hour = 3

  disk_size          = 10
  disk_type          = "PD_SSD"
  activation_policy  = "ALWAYS"
  shared_vpc_context = true
  database_flags     = [{ name = "max_connections", value = "250" }]
  user_labels = local.env_vars.locals.labels
  deletion_protection            = true
  point_in_time_recovery_enabled = false
  # network
  ip_configuration = {
    private_network     = "projects/network-project/global/networks/my-network" 
    authorized_networks = []
    ipv4_enabled        = false
    require_ssl         = null
    allocated_ip_range  = null
  }

  # Databases
  enable_default_db = false
  additional_databases = [
    {
      name      = "my_db",
      charset   = "UTF8",
      collation = "en_US.UTF8"
    }
  ]

  # Users Config
  enable_default_user = false

  maintenance_window_update_track = "canary"
  backup_configuration = {
    enabled : false,
    location : "eu",
    point_in_time_recovery_enabled : false,
    retained_backups : 7,
    retention_unit : null,
    start_time : "06:00",
    transaction_log_retention_days : null,
  }
}
}
```

### Terraform Version

```sh
1.6.6
```

### Additional information

google provider version : 5.12.0

**Comment by YpNo** [comment]
The error has came out from google-provider v.5.12.0. 
If we force provider to use the version < 5.12.0, it works. A better way is to mark the output as sensitive.

**Comment by aghassabian** [comment]
This should fix the issue: https://github.com/terraform-google-modules/terraform-google-sql-db/pull/579

**Comment by imrannayer** [comment]
Fixed in #579

**Comment by ivan-figarolabs-com** [comment]
Will there be a new version with this change?
